app/models/user.rb

```diff
@@ -0,0 +1,40 @@
+require 'bcrypt'
+
+class User
+  include DataMapper::Resource
+
+  attr_reader :password
+
+  has n, :peeps
+
+  property :id,               Serial
+  property :name,             String,  :required => true, :messages => {
+                                      :presence => 'Please enter your name' }
+  property :username,         String, :required => true, :unique => true,
+                                      :messages =>
+                                    { :presence => 'Please enter a username',
+                                      :is_unique => 'We already have that username' }
+  property :email,            String, :required => true, :unique => true,
+                                      :format => :email_address,
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)